wordpress-mobile/WordPress-iOS
# Add new ranges to activity log

Fixes #9747 

This PR creates a new set of ranges specific for Activity Log. The ranges managed are:

- Theme
- Plugin
- Comment
- Post
- Site
- Italic

All links should be clickable and it should open Safari. (In app handling will be addressed on a separated PR)

![activity_formatted](https://user-images.githubusercontent.com/9772967/42662630-4b86b44a-8600-11e8-9312-3b3103ca9433.png)

To test:

- Generate the types of event shown on the screenshots.
- Compare the styles from the app with the styles on calypso.
- Check that the styles match.
- Check that the links are actionable.
